### ISSUE TITLE
Module errors are not parsed

### ISSUE THREAD

**Issue by LarsDenBakker** [marvinhagemeister/errorstacks#5]
When a module failed to instantiate, the stack trace on the error thrown on the module script on chromium browser is different from errors caught in javascript code.

You can see it by going to `about:blank` using this code snippet:

```js
const script = document.createElement('script');
script.type ='module';
script.textContent = 'throw new Error("error in module");';
window.addEventListener('error', (e) => { console.log(JSON.stringify(e.error.stack)); });
document.body.appendChild(script);
```

The stack trace is:

```
Error: error in module
    at about:blank:1:7
```

This is not parsed correctly by the library.

**Comment by LarsDenBakker** [comment]
Looks like it's more general when a function name can't be created, for example:

```js
const c = () => new Error('my error msg');
const b = () => c();
const a = () => b();
console.log(a().stack)
```

Produces: 

```
Error: my error msg
    at c (<anonymous>:1:17)
    at b (<anonymous>:2:17)
    at a (<anonymous>:3:17)
    at <anonymous>:4:13
```

**Comment by marvinhagemeister** [comment]
Fix is released as part of `2.1.0` 👍

**Comment by LarsDenBakker** [comment]
Works great, thanks!
